src/build_rock/lpci_build/lpci_build.sh

```diff
@@ -0,0 +1,58 @@
+#! /bin/bash
+
+
+set -e
+
+
+function usage(){
+    echo
+    echo "$(basename "$0") -d <rockcraft directory> -c <launchpad credentials>"
+    echo
+    echo "Build local rockcraft project on Launchpad."
+    echo
+    echo -e "-d \\t Directory to rockcraft project file. "
+    echo -e "-c \\t Launchpad credentials. "
+}
+
+while getopts "c:d:" opt
+do
+    case $opt in
+        d)
+            ROCKCRAFT_DIR="$OPTARG"
+            ;;
+        c)
+            LC_CREDENTIALS_B64="$OPTARG"
```

Comment:
```suggestion
            LP_CREDENTIALS_B64="$OPTARG"
```
Also for the following occurances.

Reply:
Whoops! Thanks